playbooks/openstack-prepare_caasp_workers_for_update.yml

```diff
@@ -0,0 +1,37 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+---
+- hosts: caasp-admin:caasp-masters:caasp-workers
+  gather_facts: no
+  tasks:
+    - name: Ensure all the vars are loaded
+      include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
+
+    - name: Configure host repositories
+      zypper_repository:
+        name: "{{ item }}"
+        repo: "{{ deploy_on_openstack_repos_to_configure[item] }}"
+        autorefresh: True
+        auto_import_keys: yes
+        state: present
+      loop: "{{ deploy_on_openstack_caasp_repos_per_imagename[deploy_on_openstack_caasp_image] }}"
+
+    - name: Perform dist-upgrade
+      command: transactional-update cleanup dup salt
```

Comment:
What happens if you do `transactional-update cleanup dup salt reboot`[0]?
Would that restart the servers when appropriate without needing to interact with velum?

[0] http://users.suse.com/~kukuk/SUSE-CaaSP-Docu/transactional-update.8.html#reboot

Reply:
My understanding was that the reboot should be done through the velum-interactions